.circleci/config.yml

```diff
@@ -210,6 +210,22 @@ jobs:
 
       - run: mix credo --only readability,consistency,warning,refactor
 
+  release:
+    docker:
+      # Ensure .tool-versions matches
+      - image: circleci/elixir:1.7.2
+        environment:
+          MIX_ENV: dev
+
+    working_directory: ~/app
+
+    steps:
+      - attach_workspace:
+          at: .
+
+      - run: mix release
+
```

Comment:
We should add the release tar.gz as an artifact so it's stored by CircleCI

Reply:
It would make sense to pack the artefacts + logs once we have acceptance test run against it.